src/renderer/components/QueryList/QueryList.css

```diff
@@ -18,6 +21,31 @@
   cursor: pointer;
 }
 
+.QueryList-new .QueryList-filter {
```

Comment:
UI style is copied from "Data source" filtering field 😀
![Group 2809](https://user-images.githubusercontent.com/4290598/209770884-42776f0f-22be-40d2-9505-ec08dfc6b298.png)